pyproject.toml

```
@@ -234,7 +234,9 @@ exclude = [
   "lib/matplotlib/tests/",
   # tinypages is used for testing the sphinx ext,
   # stubtest will import and run, opening a figure if not excluded
-  ".*/tinypages"
+  ".*/tinypages",
+  # pylab's imports cause failures since numpy 2.2
+  "lib/matplotlib/pylab.py"
```

Optional: If we want to give context, we can be more specific:

```suggestion
  # pylab's numpy wildcard imports cause re-def failures since numpy 2.2
  "lib/matplotlib/pylab.py",
```
Also, included a comma at the end, so that the next addition to the list does not change the pylab line.

---

You also need to add an exception to the stubtests, which seem to be configured separately at the top of the exclude list.